custom/enikshay/tests/test_voucher_counts.py

```diff
@@ -149,3 +149,36 @@ def test_prescription_refill_missing_days(self):
             {},
             EpisodeVoucherUpdate(self.domain, self.cases['episode']).get_prescription_refill_due_dates()
         )
+
+    def test_voucher_generation_dates(self):
+        prescription1 = self.create_prescription_case({
+            'drugs_ordered_readable': "Happy Pills, Sad Pills, Buggy Pillz"
+        })
+        voucher11 = get_voucher_case_structure(None, prescription1.case_id, {
+            'date_fulfilled': '2013-01-01',
+            'date_issued': '2012-01-01',
+            'state': "available",
+        })
+        voucher21 = get_voucher_case_structure(None, prescription1.case_id, {
+            'date_fulfilled': '2013-01-01',
+            'date_issued': '2012-01-01',
```

Comment:
Both vouchers have the same `"date_issued"`. It might improve the test a little bit if these were different values.